system/metrics/sinks/elastic/elasticSink.hpp

```diff
@@ -18,6 +18,22 @@
 #include "jptree.hpp"
 #include "jstring.hpp"
 
+//including cpp-httplib single header file REST client
```

Comment:
A better solution would be to move the definition of the ElasticMetricSink class into the c++ file.  If this was included from many places, then each of those places would now be processing the entire header file.